nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -14,10 +14,6 @@ In addition to numerous new and upgraded packages, this release has the followin
   database updater from MaxMind. Available as
   [services.geoipupdate](options.html#opt-services.geoipupdate.enable).
 
-* [sourcehut](https://sr.ht), a collection of tools useful for software
-  development. Available as
-  [services.sourcehut](options.html#opt-services.sourcehut.enable).
-
```

Comment:
Probably didn't intend to remove this, right?

Reply:
ah sorry i messed up stuff